### DISCUSSION TITLE
Regarding Example code

### DISCUSSION THREAD

**Discussion by veekster** [pymodbus-dev/pymodbus#1036] [Q&A] [answered]
I have version 2.5.3 installed.  Does the published example code work with this version or is it for newer bleeding edge version(s)?
In particular I'm trying to understand and use the async client examples, and it seems my import statement has to be thus to work:
from pymodbus.client.asynchronous.async_io import AsyncioModbusTcpClient
I'm using python 3.9.2

**Comment by veekster** [comment]
P.S. I'm referring to example code I'm seeing in ReadTheDoc, latest.

**Comment by janiversen** [comment, marked as answer]
The example code is in transit so some of it still work with 2.5.3

If you clone the repository and checkout 2.5.3 the examples directory will be correct for 2.5.3

**Comment by veekster** [comment]
Thank you!
